Character/Classes/Warlock/Warlock.xml

```diff
@@ -1742,7 +1742,7 @@
 			</feature>
 		</autolevel>
 		<!-- Unearthed Arcana Eldritch Invocations-->
-		<autolevel level="2">
+		<autolevel level="2" IL="IL">
```

Comment:
Same as 5

Reply:
btw, UA invocations are not repeated each next level, which is incorrect and inconsistent with how original invocations work.